tests/e2e-pw/tests/blocks/cart/merchant.spec.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { editBlockPage } from '../../../utils/navigation';
+import { BlockTestingProperties } from '../../../types';
+import { openGlobalBlockInserter } from '../../../utils/editor';
+
+const blockProperties: BlockTestingProperties = {
+	title: 'Cart',
+	slug: 'woocommerce/cart',
+	selectors: {
+		block: '.wp-block-woocommerce-cart',
+		insertButton: "//button//span[text()='Cart']",
+	},
+};
+
+test.describe( 'Merchant → Cart', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.beforeEach( async ( { page } ) => {
+		await editBlockPage( page, blockProperties );
+	} );
+
+	test( 'it renders without crashing', async ( { page } ) => {
+		const blockPresence = page.locator(
+			blockProperties.selectors.className
```

Comment:
Above, I see the following constant:
```tsx
const blockProperties: BlockTestingProperties = {
	title: 'Cart',
	slug: 'woocommerce/cart',
	selectors: {
		block: '.wp-block-woocommerce-cart',
		insertButton: "//button//span[text()='Cart']",
	},
};
```
This definition does not have the selector `className`. What about if we change the constant above to
```tsx
const blockProperties: BlockTestingProperties = {
	title: 'Cart',
	slug: 'woocommerce/cart',
	selectors: {
		className: '.wp-block-woocommerce-cart',
		insertButton: "//button//span[text()='Cart']",
	},
};
```

While we could also change `blockProperties.selectors.className` to `blockProperties.selectors.block`, I believe using `className` increases the readability of the code.